argilla-server/src/argilla_server/contexts/datasets.py

```diff
@@ -936,7 +936,9 @@ async def create_response(
         )
 
         await db.flush([response])
+        await distribution.refresh_record_status(db, record, autocommit=False)
         await _touch_dataset_last_activity_at(db, record.dataset)
+        await search_engine.index_records(record.dataset, [record])
```

Comment:
We need to do a partial update of the record to only refresh the `status` in elasticsearch but in the meantime we can do this:

```suggestion
        await _preload_records_relationships_before_index(self._db, record)
        await search_engine.index_records(record.dataset, [record])
```